app/src/main/java/to/dev/dev_android/view/main/view/MainActivity.kt

```diff
@@ -43,8 +44,10 @@ class MainActivity : BaseActivity<ActivityMainBinding>(), CustomWebChromeClient.
     private fun setWebViewSettings() {
         binding.webView.settings.javaScriptEnabled = true
         binding.webView.settings.domStorageEnabled = true
-        binding.webView.webViewClient = CustomWebViewClient(this@MainActivity, binding)
-        binding.webView.webChromeClient = CustomWebChromeClient(BuildConfig.baseUrl, binding, this)
+        binding.webView.webViewClient = CustomWebViewClient(this@MainActivity){
```

Comment:
A linter like `Ktlint` would complain here:
```suggestion
        binding.webView.webViewClient = CustomWebViewClient(this@MainActivity) {
```
Anyway looks good to me :)